Assets/Scripts/StringmoreCamera/FollowCamera.cs

```diff
@@ -110,6 +112,11 @@ public void ExitFixCameraTrigger(FixCameraTrigger trigger)
             _fixCameraTriggers.Remove(trigger);
         }
 
+        public void SetShakeOffset(Vector3 offset)
+        {
+            shakeOffset = offset;
+        }
```

Comment:
nit: could use auto-property like `public Vector3 ShakeOffset {private get; set;}` to avoid writing setters manually